packages/tupaia-web/src/layout/Sidebar/StaticMap.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import polyline from '@mapbox/polyline';
+import { Position } from 'geojson';
+import { DEFAULT_BOUNDS } from '../../constants';
+import { Media } from './Media';
+
+const areBoundsValid = (b: Position[]) => {
+  return Array.isArray(b) && b.length === 2;
+};
+
+const getLatLongForBounds = (bounds: Position[]) => {
+  const lats = bounds.map(latLong => latLong[0]);
+  const longs = bounds.map(latLong => latLong[1]);
+  const minLat = Math.min(...lats);
+  const maxLat = Math.max(...lats);
+  const minLong = Math.min(...longs);
+  const maxLong = Math.max(...longs);
+
+  const latitude = minLat + (maxLat - minLat) / 2;
+
+  let longitude = minLong + (maxLong - minLong) / 2;
+
```

Comment:
This is pedantic, but I wonder if we can DRY this up with a reusable function, since these essentially do the same thing? Could accept an array of coords, make it calculate the min, max, and actual value, and then the longitude could do something beyond that with the rest of the logic, if necessary.